src/index.js

```diff
@@ -41,17 +41,16 @@ function _printNode(node, lvl, indent) {
 }
 
 
-export default function({types: t}) {
-
+export default function({types: t, version}) {
   /*
     Babel 7 renamed RestProperty to RestElement.
-    Check which one is available, then make a copy of it for future references.
-    Prefer the older version when both exist for babel 6 compatibility
+    Check the Babel version number supplied to the plugin
+    and return the result of the preferred function call.
   */
   const isRestElement = (tx, node, opts) => 
-    tx.isRestElement && !tx.isRestProperty ? 
-        tx.isRestElement(node, opts) :
-        tx.isRestProperty(node, opts);
+    version.startsWith(7)
```

Comment:
Would it make sense to do a greater-than check instead of a equality check here? That might future-proof against regressions if/when Babel 8 comes out.

Maybe something like?: 
```
const majorVersion = Number(version.split('.')[0]);
majorVersion >= 7 ? ... : ...;
```